### DISCUSSION TITLE
AnimatedPNG(APNG)

### DISCUSSION THREAD

**Discussion by feo52** [increments/qiita-discussions#408] [要望・アイデア・不具合]
下書きのプレビューではアニメーションしますが、
投稿をすると該当画像がアニメーションしなくなります。
これは不具合でしょうか?
それとも仕様でしょうか?

**Comment by tanaka-asami415** [comment]
@feo52 
ご報告ありがとうございます！

恐れ入りますが、APNGには対応していないため、
投稿後の挙動（アニメーションしない）の方が正しい仕様となります。

下書きで動くことは把握していなかったのですが、軽く調べてみたところ
記事ページでは imgix 経由で画像を配信しておりますが、
その際 imgix で アニメーションしない画像に変換されています。

下書きのプレビューでは imgix 経由ではないため、アニメーションします。
ご不便をおかけして申し訳ないですが、ご了承いただけますと幸いです🙇

**Reply by yoshi389111** [reply]
本件ですが、Qiitaヘルプの [画像のアップロード方法](https://help.qiita.com/ja/articles/qiita-image-upload) あたりに、注記があるとわかりやすいとおもうのですが、いかがでしょうか。

**Comment by feo52** [comment]
ご回答ありがとうございます。
ご回答内容承知いたしました。
